Algorithms/EuclidGCD.py

```diff
@@ -0,0 +1,13 @@
+# Uses python3
+import sys
+
+def gcd_euclid(a, b):
+	if a == 0 :       #base case
+		return b
+
```

Comment:
Don't leave line empty here.

Put the function with return statement together.